gaitutils/emg.py

```diff
@@ -104,7 +104,7 @@ def _edf_export(self, filename):
         # default to EDF+ for the time being
         file_type = pyedflib.FILETYPE_EDFPLUS
         f = pyedflib.EdfWriter(
-            filename,
+            str(filename),
```

Comment:
I'm a bit curious about the purpose of this - `filename` should already be a `str` instance right?

Reply:
At least on my laptop it is PosixPath or something like that.

Reply:
Ok, I get it now. I started using pathlib at some point, but pyedflib (like many other libraries) still does not accept pathlib objects. Should have written a unit test...

Reply:
There is a unit test. But it's only activated when pyedflib is installed. Another reason to make it a normal dependency.

Reply:
good point